pandas/tests/series/test_missing.py

```diff
@@ -883,7 +883,9 @@ def test_interpolate_akima(self):
                           index=Index([1.0, 1.25, 1.5, 1.75,
                                        2.0, 2.25, 2.5, 2.75, 3.0]))
         # interpolate at new_index
-        new_index = ser.index.union(Index([1.25, 1.5, 1.75, 2.25, 2.5, 2.75]))
+        new_index = ser.index.union(
+            Index([1.25, 1.5, 1.75, 2.25, 2.5, 2.75])
```

Comment:
why do you need to do this?

Reply:
`ser.index` is a `RangeIndex`. `Index([1.25..., 2.75])` is a `Float64Index`. Due to not being compatible they produce an object dtype Index, which then needs to be cast back to float.

This operation is not what was being checked in this test, and was just a matter of the mechanics of the new behavior to get the actual check to work.